exporter/trace/otelcol/README.md

```diff
@@ -0,0 +1,24 @@
+# OpenTelemetry Collector Go Exporter
```

Comment:
Did you consider naming this "otlp"? It seems like the term we've used for this protocol everywhere.

Reply:
done